src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -78,6 +78,15 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     config.clearHelmRelease = true;
   }
 
+  void enableCleanupOfHelmReleaseOnFailure() {
+    config.clearHelmReleaseOnFailure = true;
+  }
+
+  void enableCleanupOfHelmReleaseAlways() {
+    config.clearHelmRelease = true;
```

Comment:
Should we rename it ?
```suggestion
    config.clearHelmReleaseOnSuccess = true;
```

Reply:
I don't want to change things that were already there, if at all possible. Not as part of this ticket anyway.

I'd like this ticket to be laser-focused to solving the issue at hand. Once that's done, I'll be more than happy to do some minor improvements here and there whenever I get the time.

Reply:
Hmm, I don't think it's unrelated. We added a new var `clearHelmReleaseOnFailure` , so it makes sense to refactor existing var to make it more meaningful. Nothing should break for changing that.

Reply:
Not unrelated, just not necessary given the scope of this ticket. I merely preferred to separate the feature from the refactoring, as it keeps the PR smaller and therefore easier to understand.

Anyway: alright. Changed it there and in all places where it's used in this repo.